### PR TITLE
Updated e2e tests on travis

### PR DESCRIPTION
**Pull request by ianlewis**
- Updated tests to run using Go 1.14
- Added test for containerd 1.3
- Updated release of runsc to test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-  - GO111MODULE=on CONTAINERD_VERSION=1.1.5 RUNSC_VERSION=2019-08-06 TEST=untrusted-workload
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=untrusted-workload
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=runtime-handler
-  - GO111MODULE=on CONTAINERD_VERSION=1.2.1 RUNSC_VERSION=2019-08-06 TEST=runtime-handler-shim-v2
-
+  - CONTAINERD_VERSION=1.1.5 RUNSC_VERSION=release-20200219.0 TEST=untrusted-workload
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=untrusted-workload
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
+  - CONTAINERD_VERSION=1.2.13 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
```
Comment (kevinGC) [inline .travis.yml]: This line is repeated.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+  - CONTAINERD_VERSION=1.3.3 RUNSC_VERSION=release-20200219.0 TEST=runtime-handler-shim-v2
```
Comment (kevinGC) [inline .travis.yml]: I'm probably just missing information here, but why only run the `runtime-handler-shim-v2` test and not `runtime-handler` or `untrusted-workload`?

File: .travis.yml (hunk 1 above)
Comment (ianlewis) [reply .travis.yml]: Done

File: .travis.yml (hunk 2 above)
Comment (ianlewis) [reply .travis.yml]: I added shim v1 (runtime-handler). I think that's what I meant to do in the first place.
The untrusted workload annotation isn't supported anymore.

**Comment by ianlewis** [comment]
Getting errors because containerd now depends on libseccomp 2.4
```
containerd: symbol lookup error: containerd: undefined symbol: seccomp_api_set
```
https://github.com/containerd/containerd/issues/4008

**Comment by ianlewis** [comment]
Had more issues with the cri grpc port already being open for containerd 1.1. That took entirely too long to fix. Should be cleared up now though.

**Pull request merged**